### PR TITLE
Add additional information to the Yext integration documentation

### PR DESCRIPTION
**Pull request by bschoeneweis**
## What?
Additional information and updates for the Yext integration.

## Screenshots (optional)
image

### CODE REVIEW

**Comment by bschoeneweis** [comment]
## Updated Screenshots
Screen Shot 2021-12-06 at 11 51 11 AM

Screen Shot 2021-12-06 at 11 51 21 AM

File: docs/integrations/yext.mdx [hunk 1]
```diff
@@ -8,8 +8,39 @@ The Yext integration is available on the [Enterprise](https://radar.io/pricing)
 
 ## Configuration
 
-On the Radar [Integrations page](/integrations) under *Yext*, click *Connect*. You will be redirected to Yext, where you can authorize Radar. If authorization succeeds, you will be redirected to the Radar [Geofences page](/geofences). If authorization fails, you will be redirected back to the Radar [Integrations page](/integrations).
+On the Radar [Integrations page](/integrations) under *Yext*, click *Enabled* to *Yes* and optionally configure any additional settings.
```
Comment (jsani-radar) [inline docs/integrations/yext.mdx:11]: nit: let's change this to "set *Enabled* to *Yes*" to match the copy for other integrations

File: docs/integrations/yext.mdx [hunk 2]
```diff
@@ -8,8 +8,39 @@ The Yext integration is available on the [Enterprise](https://radar.io/pricing)
...
+### Additional Settings
+You can choose to override certain defaults with additional configuration settings for the [geofences](/documentation/geofences) synced from Yext.
+
+The available overrides include:
+- Default geofence `type`
+- Default geofence `radius`
+- An override for the mapping of the `description` field to a different Yext field than `locationName` (e.g. `address`)
+- An override for the default `tag` value that will be mapped to a Yext field (e.g. `schemaTypes.0`)
+
+Additionally, you can map up to 16 Yext fields to Radar geofence metadata. Use the *dot notation* to access nested Yext fields.  For example, if you have a Yext entity with the following property: `"schemaTypes": ["Restaurant"]`, you can store the value `"Restaurant"` by adding `schemaTypes.0` as a *Yext Key*.  If you include a Yext field in the metadata mapping and that field is not found, or the value in Yext is undefined or `null`, the value will not be copied over to the geofence.
```
Comment (jsani-radar) [inline docs/integrations/yext.mdx:39]: take out "the" so the second sentence reads "Use *dot notation*..."

Comment (jsani-radar) [reply docs/integrations/yext.mdx:39]: also, are we referring to Yext objects as "locations" or "entities" - we use "location" in the table above but "entity" here, let's standardize on one

File: docs/integrations/yext.mdx [hunk 3]
```diff
@@ -8,8 +8,39 @@ The Yext integration is available on the [Enterprise](https://radar.io/pricing)
...
+
+### Syncing Data
+Once configured, click *Connect & Sync* and you will be redirected to Yext, where you can authorize Radar. Then, you will be redirected back to the Radar [Integrations page](/integrations) and there will be an indication if the authorization succeeded or failed.  If the authorization succeeded, your Yext locations will immediately begin syncing to Radar [geofences](/documentation/geofences).
```
Comment (jsani-radar) [inline docs/integrations/yext.mdx:42]: nit: let's have the second sentence read "Then, you will be redirected back to the Radar Integrations page with an indication of whether the authorization succeeded or failed" - slightly less wordy

File: docs/integrations/yext.mdx [hunk 4]
```diff
@@ -8,8 +8,39 @@ The Yext integration is available on the [Enterprise](https://radar.io/pricing)
...
+
+Radar will automatically keep geofences in sync when you create or update locations in Yext.  Radar will sync geofences both to the *Test* and *Live* environments.  **If you update the geometry of any geofences synced with Yext, your changes will not be overwritten with subsequent updates.**
```
Comment (jsani-radar) [inline docs/integrations/yext.mdx:44]: does this mean updating the geometry in Radar or in Yext? let's make this a bit clearer

File: docs/integrations/yext.mdx [hunk 5]
```diff
@@ -8,8 +8,39 @@ The Yext integration is available on the [Enterprise](https://radar.io/pricing)
...
+
+You are able to update any of the defaults after the initial sync by changing the integration settings and clicking *Update & Sync*.  Once clicked, a full sync of every Yext location will occur.  **A change in these default overrides will only be reflected for newly created locations in Yext.  Previously synced locations turned geofences will not be updated to the new defaults.**
```
Comment (jsani-radar) [inline docs/integrations/yext.mdx:46]: Let's remove "You are able to" in the beginning here so it reads - "Update any of the defaults after the initial sync by changing..."

**Review by jsani-radar — COMMENTED** [review]
few small comments on copy changes to make it a bit more clear and less wordy

will be 👍🏽 on next pass

File: docs/integrations/yext.mdx (hunk 2 above)
Comment (bschoeneweis) [reply docs/integrations/yext.mdx:39]: Good call, I'll standardize  to "location"

File: docs/integrations/yext.mdx (hunk 4 above)
Comment (bschoeneweis) [reply docs/integrations/yext.mdx:44]: How's this?

Radar will automatically keep geofences in sync when you create or update locations in Yext.  Radar will sync geofences both to the *Test* and *Live* environments.  **If you update the geometry of a Radar geofence synced with Yext, your changes will not be overwritten in Radar with subsequent updates.**

Comment (jsani-radar) [reply docs/integrations/yext.mdx:44]: nice, I like that phrasing!

**Review by jsani-radar — APPROVED** [review]
lgtm! 🚀

**Pull request merged**